File: create_smoketest_structure.py
import os
import sys

basePath = r'C:\Users\KristinaDenney\source\repos\glyphor-ai-company'

# Create directories
dirs = [
    r'packages\smoketest',
    r'packages\smoketest\src',
    r'packages\smoketest\src\layers',
    r'packages\smoketest\src\utils'
]

print('Creating directories...')
for dir in dirs:
    fullPath = os.path.join(basePath, dir)
    try:
        os.makedirs(fullPath, exist_ok=True)
        print(f'✓ Created: {dir}')
    except Exception as e:
        print(f'✗ Failed to create {dir}: {e}')
        sys.exit(1)

print('\nDirectory structure created successfully!')

File: create_smoketest_structure.js
const fs = require('fs');
const path = require('path');

const basePath = 'C:\\Users\\KristinaDenney\\source\\repos\\glyphor-ai-company';

// Create directories
const dirs = [
  'packages\\smoketest',
  'packages\\smoketest\\src',
  'packages\\smoketest\\src\\layers',
  'packages\\smoketest\\src\\utils'
];

console.log('Creating directories...');
dirs.forEach(dir => {
  const fullPath = path.join(basePath, dir);
  if (!fs.existsSync(fullPath)) {
    fs.mkdirSync(fullPath, { recursive: true });
    console.log(`✓ Created: ${dir}`);
  } else {
    console.log(`✓ Already exists: ${dir}`);
  }
});

console.log('\nDirectory structure created successfully!');
